src/stylesheets/components/_tabs.scss

```diff
@@ -9,6 +9,11 @@
   @include mq($until: desktop) {
     display: none;
   }
+
+  .govuk-prose-scope & { // A specific selector to reset .govuk-prose-scope > ul
```

Comment:
To avoid confusion, suggest removing the `>`s from the comment here and below as it looks like we're resetting against a selector that uses direct descendents.

Reply:
Good point. Removed `>` from comments.